src/race.c

```diff
@@ -271,6 +271,24 @@ void ToggleRace(void)
 	apply_race_settings();
 }
 
+void RaceCountdownChange(float t)
+{
+	float rcd = cvar("k_race_countdown") + t;
+
+	if (match_in_progress || !isRACE() || race_is_started())
+        {
+		return;
+	}
+
+	if (rcd < 6 && rcd > 0)
```

Comment:
Please use full bracket syntax, to match the code style:
`if ((rcd < 6) && (rcd > 0))`